pkg/json.go

```diff
@@ -196,80 +168,16 @@ func (r *ScorecardResult) AsJSON2(showDetails bool,
 		}
 		out.Checks = append(out.Checks, tmpResult)
 	}
+
 	if err := encoder.Encode(out); err != nil {
 		return sce.WithMessage(sce.ErrScorecardInternal, fmt.Sprintf("encoder.Encode: %v", err))
 	}
 
 	return nil
 }
 
-func (r *ScorecardResult) AsSJSON(showDetails bool,
+func (r *ScorecardResult) AsFJSON(showDetails bool,
 	logLevel log.Level, checkDocs docs.Doc, writer io.Writer,
 ) error {
-	score, err := r.GetAggregateScore(checkDocs)
```

Comment:
removing this function for now as we focus on the "finding" format first instead of "structured" format.